deploy/docker/cp-dts-log-checker/dts-log-checker.py

```diff
@@ -0,0 +1,201 @@
+email_template_file = os.path.join(dts_path, DTS_EMAIL_TEMPLATE)
+pattern = os.environ.get('CP_DTS_LOG_MESSAGE_PATTERN', r'/*ERROR./*')
+run_id = os.getenv('RUN_ID', default='0')
+pipeline_name = os.getenv('PIPELINE_NAME', default='pipeline')
+runs_root = os.getenv('CP_RUNS_ROOT_DIR', default='/runs')
+run_dir = os.getenv('RUN_DIR', default=os.path.join(
+    runs_root, pipeline_name + '-' + run_id))
+log_dir = os.getenv('LOG_DIR', default=os.path.join(run_dir, 'logs'))
+pipeline_api = os.getenv('API')
+api = PipelineAPI(api_url=pipeline_api, log_dir=log_dir)
+logger = RunLogger(api=api, run_id=run_id)
+logger = TaskLogger(task='DTSLogCheck', inner=logger)
+logger = LocalLogger(inner=logger)
+
+
+def get_email_body(email_template_file):
+    body = {}
+    if os.path.exists(email_template_file):
+        with codecs.open(email_template_file,  "r", "utf-8") as html_file:
+            body = html_file.read()
+    else:
+        body = """<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+    <html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <style>
+            table,
+            td {{
+                border: 1px solid black;
+                border-collapse: collapse;
+                padding: 5px;
+            }}
+        </style>
+    </head>
+    <body>
+        <p>Dear user,<br>
+        *** This is a system generated email, do not reply to this email ***<br>
+        <b>DTS log files with errors:</b><br>
+        {}<br>
+        Best regards, 
+        EPM-AWS-Dev Platform<br>
+    </body>
+    </html>
+        """
+
+    return body
+
+def generate_table(files_with_errors):
+
+    def get_table_header(dts_log_url):
+        if dts_log_url:
+            return """
+                <tr>
+                    <td><b>Files</b></td>
+                    <td><b>Access link</b></td>
+                </tr>"""
+        else:
+            return """
+                <tr>
+                    <td><b>Files</b></td>
+                </tr>"""
+
+
+    def get_table_row(dts_log_url, file):
+        if dts_log_url:
+            return "<td>{}</td>".format(file) + "<td>{}</td>".format('<a href=' + '"' + str(dts_log_url) + os.path.basename(file) + '&contentDisposition=ATTACHMENT' + '"' + '>Link</a')
+        else:
+            return "<td>{}</td>".format(file)
+    
+    table = "<table>" + get_table_header(dts_log_url)
+    for file in files_with_errors:
+        table += "<tr>" + get_table_row(dts_log_url, file) + "</tr>"
+    table += "</table>"
+    return table
+
+
+
+
+def retrieve_timestamp(str):
+    match_str = re.search(
+        '(\d{4})-(\d{2})-(\d{2}) (\d{2}):(\d{2}):(\d{2}).(\d{3})', str)
+    if match_str is not None and match_str.group():
+        return datetime.strptime(match_str.group(), "%Y-%m-%d %H:%M:%S.%f")
+    else:
+        logger.info('There is no applicable date')
```

Comment:
it should be at least `warn`
and also lets change message to `can't find an applicable date in string: {}`.format(str)